enum/tarifart/tarifart.go

```diff
@@ -13,4 +13,5 @@ const (
 	MEHRTARIF                  // MEHRTARIF = multiple (>2) tariffs
 	SMARTMETER                 // SMARTMETER means Smart Meter Tariff
 	LEISTUNGSGEMESSEN          // LEISTUNGSGEMESSEN means Leistungsgemessener Tarif
+	SMART_METER
```

Comment:
https://github.com/Hochfrequenz/BO4E-dotnet/blob/338cf40a943d051f50edcf387769e867401e787a/BO4E/ENUM/Tarifart.cs#L27-L28

`SMART_METER` is the way we expect the enum to be.